pkg/sql/codegen/attribute/attribute.go

```diff
@@ -158,6 +160,18 @@ func NewDictionaryFromModelDefinition(estimator, prefix string) Dictionary {
 // PremadeModelParamsDocs stores parameters and documents of all known models
 var PremadeModelParamsDocs map[string]map[string]string
 
+// ExtractDocString extracts parameter documents from python doc strings
+func ExtractDocString(module ...string) {
+	cmd := exec.Command("python", "-uc", fmt.Sprintf("__import__('extract_docstring').print_param_doc('%s')", strings.Join(module, "', '")))
+	output, e := cmd.CombinedOutput()
+	if e != nil {
+		log.Println("ExtractDocString failed: ", e, string(output))
+	}
+	if e := json.Unmarshal(output, &PremadeModelParamsDocs); e != nil {
```

Comment:
Maybe add a comment that unmarshal can append entries rather than overwrite it: https://golang.org/pkg/encoding/json/#Unmarshal